src/test/java/cloudBankPages/ClientListPage.java

```diff
@@ -0,0 +1,35 @@
+package cloudBankPages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import static org.testng.Assert.assertTrue;
+
+public class ClientListPage {
+
+    WebDriver driver;
+    @FindBy(xpath = "//h1[text()='Client List']")
+    public WebElement clientListTitle;
+
+    public WebElement getAccountNumLocator(String text){

```

Comment:
Give the full spelling in the method name. Ex. getAccountNumberLocator.

Reason: It is better to be more specific and descriptive to eliminate any ambiguity.

Reply:
Also, this method returns the WebElement, not just a locator. So a more accurate method name would be: getAccountNumberElement.

Apply this to the rest of your methods.

Reply:
done